pynblint/notebook.py

```diff
@@ -44,6 +46,10 @@ def __init__(self, path: Path):
             self.ast = ast.parse(self.script)
         except SyntaxError:
             self.has_invalid_python_syntax = True
+        self.imported_package = self._get_import_package_set()
+        #  non posso accedere ai set creati nella classe Repository,
+        #  come dovrei procedere?
+        # self.missing_requiremet = self.imported_package.difference(Repository.)
```

Comment:
This check should be performed within a dedicated linting function (inside `nb_linting.py`, I guess).
Here we are just defining notebook properties; the notebook property that we need to add in order to make the check feasible is the set of imported packages, as you already do at line 49.